### ISSUE TITLE
error in anonympy setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.6*'

### ISSUE THREAD

**Issue by gianmarcoaversanoenx** [ArtLabss/open-data-anonymizer#26]
```
pip install anonympy
```

retults in

```
error in anonympy setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.6*'
```

this error happens just sometimes... i am using python 3.10.10 and i can install it, then i suddenly cannot anymore.

**Comment by dannkunt** [comment]
Same on python 3.11

**Comment by ajouanjean** [comment]
same here

**Comment by shukkkur** [comment]
@ajouanjean Pull requested merged now

**Comment by LSD-98** [comment]
Hello,
I am quite new to Python and Git, but I still have the same error when running
``` pip install anonympy ```. Here is the full output:

```
pip install anonympy
Collecting anonympy
  Using cached anonympy-0.3.7.tar.gz (5.8 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in anonympy setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.6*'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.
```

What I do not understand is that the ```setup.py``` file from the tar.gz folder downloaded on pypi is not updated and still has the ```*```. The cape_privacy fix is also missing on pypi.

Edit : fixed it by installing the package from the repository. ```pip install git+https://github.com/ArtLabss/open-data-anonymizer.git```

**Comment by ajouanjean** [comment]
Yes, files in PyPI have not yet been updated, so using `pip` will not work unfortunately.
@shukkkur could you please push a new version to pypi?
Thanks a lot

**Comment by shukkkur** [comment]
Hey @ajouanjean 

I am very happy that people are using our package. Will definitely do this weekend, although I am totally clueless how to do did I do it in the past 😅 

Thanks for the reminders and forgive my laziness :)

**Comment by keenborder786** [comment]
I am still facing the same error and will be happy to fix it.

**Comment by djmhunt** [comment]
Hi @shukkkur,

It is great to see that the GitHub code has been updated. Do you have any update on when you plan to create a new release here or on pypi?

**Comment by filipafcastro** [comment]
I'm also having trouble now with installing the package from the repository. 
![image](https://github.com/ArtLabss/open-data-anonymizer/assets/10077981/61033284-8357-470a-885b-d60164ee137c)

> Error downloading object: anonympy/tests/pdf/expected/Tesseract-OCR/iconv.dll (806b29c): Smudge error: Error downloading anonympy/tests/pdf/expected/Tesseract-OCR/iconv.dll (806b29c2da610a1f8145dc7934f779dd7a569e4a95beac6433d72a833b01feac): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Any idea what these data packs are? Thanks!

**Comment by onakatomi** [comment]
@filipafcastro I have the same issue. @shukkkur any update for when this will be fixed, or when the fixed version (latest github version) will be pushed to pypi?

**Comment by elyse-weiss** [comment]
Also, getting this issue!

**Comment by ChefKeff** [comment]
Still having this issue!
